data/actionsmap/yunohost.yml

```diff
@@ -535,6 +535,31 @@ app:
                     full: --file
                     help: Folder or tarball for upgrade
 
+        ### app_changeurl()
+        changeurl:
```

Comment:
Following the discussion about the name convention with ljf (https://dev.yunohost.org/issues/872), I think we need to revert back to `change-url` (instead of `changeurl`) (sorry for inducing you in error about that Bram :/)